src/dmd/dtemplate.d

```diff
@@ -794,6 +794,31 @@ extern (C++) final class TemplateDeclaration : ScopeDsymbol
         return result;
     }
 
+    /******************************
+     * Create a scope for the parameters of the TemplateInstance
+     * ti in the parent scope sc from the ScopeDsymbol paramsym.
+     *
+     * If paramsym is null a new ScopeDsymbol is used in place of
+     * paramsym.
+     * Params:
+     *      ti:         the TemplateInstance whose parameters to generate the scope for.
+     *      sc:         the parent scope of ti
+     *      paramsym:   the ScopeDsymbol to create the scope under
```

Comment:
Ddoc uses = for the parameters

Reply:
like `ti       = the TemplateInstance ...`? (I've never actually used ddoc).

Reply:
Yep and that's why DAutoTest (the doc tester) complained FWIW ;-)

See also: https://dlang.org/spec/ddoc.html#special_sections

Reply:
Yeah, but `Warning: Ddoc: parameter count mismatch` doesn't tell me that.